docs: change linting CLI and add custom rules (#1031)
keptn/lifecycle-toolkit: README.md

<!---x-release-please-end-->

The Lifecycle Toolkit uses the OpenTelemetry collector to provide a vendor-agnostic implementation of how to receive,
process and export telemetry data.
To install it, follow
their [installation instructions](https://opentelemetry.io/docs/collector/getting-started/).
We provide some information about this in our [observability example](./examples/support/observability/).

The Lifecycle Toolkit includes a Mutating Webhook which requires TLS certificates to be mounted as a volume in its pod.
The certificate creation
is handled automatically
by [klt-cert-manager](https://github.com/keptn/lifecycle-toolkit/blob/main/klt-cert-manager/README.md).

> Versions 0.5.0

Reviewer: this is actually valid for 0.6.0 since we postponed the release of the helm chart, would you mind correcting it ?

> Versions 0.5.0

Reviewer: ```suggestion
Versions 0.6.0
```

app.kubernetes.io/version: myAwesomeWorkloadVersion
```

In general, the Keptn Annotations/Labels take precedence over the Kubernetes recommended labels.

> If there is no version
> annotation/label and there is only one container in the pod, the Lifecycle Toolkit will take the image tag as version (

Reviewer 2: ```suggestion
If no version annotation/label exists
and the pod only contains one container, the Lifecycle Toolkit takes the image tag as version
```
